composer.json

```diff
@@ -3,6 +3,16 @@
     "type": "sylius-plugin",
     "description": "Sylius Plugin for order creation in Admin panel",
     "license": "MIT",
+    "authors": [
+        {
+            "name": "Mateusz Zalewski",
+            "homepage": "http://mpzalewski.com.pl"
```

Comment:
Don't you want to add your email also?

Reply:
Nope, all info about me is on my very advanced website 😄